docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
+
+Get all properties for a service-name (application) filtered on service-profile (profile) and service-profile (profile).
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-b& \
+  service-label=19c& \
+  service-profile=production
+```
+
+Example of data returned:
+
+```json
+[
+  {
+    "id": 7,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-name",
+    "value": "databasename-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 8,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-connection",
+    "value": "connectionstring-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>&service-profile=\<service-profile\>&property-key=\<property-key\>
+
+Get all properties for a service-name (application) filtered on service-profile (profile), service-profile (profile) and property-key (prop_key).
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-c& \
+  service-label=23.4& \
+  service-profile=secret& \
+  property-key=txeventq
+```
+
+Example of data returned.
+
+```json
+[
+  {
+    "id": 14,
+    "application": "application-c",
+    "profile": "secret",
+    "label": "23.4",
+    "propKey": "txeventq",
+    "value": "23c-kafka-name",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/property/add
+
+Create a property.
+
+```shell
+  curl -u <username>:<password> -s -X POST \
+    -d "service-name=application-d&service-label=1.0&service-profile=AI&property-key=url-to-host&property-value=hostname" \
+    http://localhost:8080/srv/config/property/add
+```
+
+Successful creation of a property returns
+
+```text
+Property added successfully.
+```
+
+#### /srv/config/property/update
+
+Update a property.
+
+```shell
+curl -u <username>:<password> -s -X PUT \
+  -d "service-name=application-d&service-label=1.0&service-profile=AI&property-key=url-to-host&property-value=new-hostname" \
+   http://localhost:8080/srv/config/property/update
+```
+
+Successful update of a property returns
+
+```text
+Property successful modified.
+```
+
+#### /srv/config/properties/delete?service-name\<service-name\>
+
+Delete all properties from a service (application).
+
+```Shell
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete? \
+  service-name=atael
+```
+
+Successful delete of properties returns:
+
+```text
+Property(ies) successfully deleted.
+```
+
+#### /srv/config/delete?service-profile=\<profile-name\>&service-profile=\<service-profile\>
+
+Delete all properties with a service profile.
```

Comment:
Change the period to a colon.

Reply:
I need change the backend for this, stay tuned